rd-cpp/src/rd_core_cpp/src/main/std/hash.cpp

```diff
@@ -0,0 +1 @@
+#include "hash.h"
```

Comment:
I'm sorry, but why do we need `hash.cpp`, `unordered_map.cpp` and `unordered_set.cpp`? I'm not a C++ expert, but aren't template classes usually header-only? What't the use of these `.cpp`? They generate no executable code by themselves, right?

Reply:
There were some gimmick with Ninja build that doesn't like it when there are solo headers.